### ISSUE TITLE
Feature: Standalone validator client

### ISSUE THREAD

**Issue by yorickdowne** [grandinetech/grandine#9]
A standalone validator client has advantages for users:
- Can use rescue node
- Can use RocketPool reverse hybrid
- Can configure a distributed setup with one VC/web3signer and multiple CL to connect to

**Comment by sauliusgrigaitis** [comment]
Yes, that's something we are working on, however, it will take a while to ship it. Grandine has a built-in VC that is coupled with BN. The code base is under heavy refactoring to decouple VC from BN, this will take some time. Until it's done the best would be to use Vouch or other VC via Grandine BN.

**Comment by KaloyanTanev** [comment]
It would be great for distributed validators (DVs) as well, where the DV plugs in as a middleware between the VC and the BN.

**Comment by Tumas** [comment]
- [x] attestations
- [x] aggregations
- [x] sync committee duties
- [x] rebase on latest glamsterdam branch
- [ ] block proposal
- [ ] ptc

**Comment by sauliusgrigaitis** [comment]
@yorickdowne could you start cooking on eth-docker side using https://github.com/grandinetech/grandine/tree/feature/separate-vc branch? It is going to be merged soon.

So now you can specify `grandine bn ...` and `grandine vc ...` if you want a separate bn/vc setup. However, we also keep the integrated approach as is for users that prefers single process. So Grandine can work both as a separate bn/vc and an integrated approach.

**Comment by yorickdowne** [comment]
@sauliusgrigaitis Is this already ePBS on the VC? That is, it's really a list of builder URLs?

```
      --builder-url <BUILDER_URL>
          External block builder URL
```

If so, adding a `--builder-min-bid` option may also be good, as would be renaming this to `--builder-urls` on the VC and introducing `--builder-urls` on the combined one. There's a question there on how to best handle the PBS->ePBS transition. You'll need the existing `--builder-url` on the CL until Glamsterdam, then `--builder-urls` on the VC takes over
